CHANGELOG.md

```diff
@@ -131,6 +131,65 @@ In addition, these existing libraries are now GA:
 
 - [Cloud Batch](/google/cloud/batch/README.md)
 
+### [BigQuery](/google/cloud/bigquery/README.md)
+
+- doc: add endpoint override snippets to generated libs ([#10129](https://github.com/googleapis/google-cloud-cpp/pull/10129))
+- feat(bigquery): add migration service ([#10034](https://github.com/googleapis/google-cloud-cpp/pull/10034))
+
+### [Bigtable](/google/cloud/bigtable/README.md)
+
+- doc(bigtable): add `*Client` samples ([#10149](https://github.com/googleapis/google-cloud-cpp/pull/10149))
+- feat(bigtable): support `GOOGLE_CLOUD_ENABLE_DIRECT_PATH` ([#9978](https://github.com/googleapis/google-cloud-cpp/pull/9978))
+
+### [IAM](/google/cloud/iam/README.md)
+
+- doc: add endpoint override snippets to generated libs ([#10129](https://github.com/googleapis/google-cloud-cpp/pull/10129))
+
+### [Pub/Sub](/google/cloud/pubsub/README.md)
+
+- doc(pubsub): samples for endpoint and auth ([#10136](https://github.com/googleapis/google-cloud-cpp/pull/10136))
+- feat(pubsub): install pubsub_mocks pkg ([#10008](https://github.com/googleapis/google-cloud-cpp/pull/10008))
+- feat(pubsub): implement blocking publisher ([#10055](https://github.com/googleapis/google-cloud-cpp/pull/10055))
+- feat(pubsub): implement per-call options for `Subscriber` ([#10043](https://github.com/googleapis/google-cloud-cpp/pull/10043))
+- fix(pubsub): limit `ModifyAckDeadlineRequest` size ([#10032](https://github.com/googleapis/google-cloud-cpp/pull/10032))
+- fix(pubsub): race condition capturing batching timer ([#9996](https://github.com/googleapis/google-cloud-cpp/pull/9996))
```

Comment:
This is a fix for the next fix, you can collapse the two.

Reply:
Done (by deleting the "fix race").